lib/jbuild

```diff
@@ -5,5 +5,20 @@
    (lwt
     lwt.unix)
   )
+  (modules vbd_store)
+ )
+)
+
+(library
+ ((name consts)
+  (modules consts)
+ )
+)
+
+(library
+ ((name local_xapi_session)
+  (modules local_xapi_session)
+  (libraries (consts lwt lwt.unix
```

Comment:
We could have the modules in their own line and sorted alphabetically, as for `vbd_store`.